packages/atomic/src/components/search/result-template-components/quickview-iframe/quickview-iframe.tsx

```diff
@@ -7,7 +7,9 @@ const writeDocument = (documentWriter: Document, content: string) => {
   documentWriter.open();
   documentWriter.write(content);
   documentWriter.close();
-  documentWriter.body.scrollTop = 0;
+  if (documentWriter.scrollingElement) {
```

Comment:
Kind of unrelated to this PR, just a bug fix for the scrollbar when a quickview load.

Not all documents will use to body to scroll (this depends on the type of document in the index), so using `scrollingElement` is a better catch all mechanism.